app/Models/LocalIndicator.php

```diff
@@ -25,4 +28,9 @@ public function globalIndicator(): BelongsTo
     {
         return $this->belongsTo(GlobalIndicator::class);
     }
+
+    public function getDomainAttribute()
+    {
+        return $this->theme->domain;
+    }
```

Comment:
Are we linking LocalIndicators to themes, or only to domains? My understanding of the LISP process is that they use the "4 themes" - which we are calling domains to avoid clashes with the longer list of themes that come from the HOLPA paper / global indicators. 

If this is right, then the teams will upload a set of indicators for each 'domain', and we won't know what 'theme'  they are linked to, so we can't have `theme()` as a relationship here - instead we could have `domain` as a new Model and make local indicators belong to a domain?